lib/ansible_tower_client/base_model.rb

```diff
@@ -37,6 +37,11 @@ def initialize(api, json_or_hash)
       super(raw_hash)
     end
 
+    def self.create(api, attributes)
+      response = api.post("#{self.endpoint}/", attributes).body
```

Comment:
The bot should complain about this redundant `self`.  The config was just added, you may need to rebase the PR.